src/frontends/tensorflow_common/src/op/conj_transpose.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2023 Intel Corporation  
+// SPDX-License-Identifier: Apache-2.0  
+//  
+
+#include "helper_ops/complex_type_mark.hpp"  
+#include "openvino/op/transpose.hpp"
+#include "openvino/op/negative.hpp"
+#include "openvino/op/concat.hpp"
+#include "common_op_table.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/shape_of.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_conj_transpose_op(const NodeContext& node){
+	default_op_checks(node, 2, {"ConjugateTranspose"});
+
+	auto x = node.get_input(0);
+	auto perm = node.get_input(1);
+	
+	
+	auto complex_type_mark = as_type_ptr<ComplexTypeMark>(x.get_node_shared_ptr());
+	if (complex_type_mark)
+	{
+		element::Type complex_part_type = complex_type_mark->get_complex_part_type();
+		auto x = complex_type_mark->input_value(0);
+
+		auto real_index = make_shared<v0::Constant>(element::i32, Shape{}, 0);
+		auto imag_index = make_shared<v0::Constant>(element::i32, Shape{}, 1);
+
+		auto gather_axis = make_shared<v0::Constant>(element::i32, Shape{1}, -1);
+		
+		auto real = make_shared<v8::Gather>(x, real_index, gather_axis)->output(0);
+		auto imag = make_shared<v8::Gather>(x, imag_index, gather_axis)->output(0);
+
+		imag = make_shared<v0::Negative>(imag);
+	
```

Comment:
please fix code-style according to clang format.

Reply:
Hello @rkazants, I fixed it.